src/pow.h

```diff
@@ -30,6 +30,10 @@ int64_t GetFilteredTime(const CBlockIndex* pindexLast);
 
 unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHeader *pblock);
 
+/** Check whether a block's work target is no more than twice and not
+ ** less than half the past the work target of the past 12 blocks */
```

Comment:
This sentence looks broken. Either way, I don't understand it.

Reply:
Maybe "Verify that" instead of "Check whether". Is that the source of the confusion, or is it still unclear?

Reply:
It's the last part that I don't understand; "less than half the past the work target"?

Reply:
New text is:

> Verify that a block's work target is within the range of half to twice the targets of the past 12 blocks.

I hope this is clear now.

Reply:
Maybe using numbers would make it more clear ("0.5x to 2x the targets of the past 12 blocks"). See Slack.